lib/Conversion/TorchToTosa/TorchToTosa.cpp

```diff
@@ -3793,14 +3793,24 @@ class ConvertAtenPoolingBaseOp : public OpConversionPattern<AtenOpT> {
 
   static int64_t getOutputDim(int64_t inputDim, int64_t kernelDim,
                               int64_t stride, int64_t padBefore,
-                              int64_t padAfter, int64_t dilation) {
+                              int64_t padAfter, int64_t dilation,
+                              bool ceilMode=false) {
     if (inputDim == kUnknownSize) {
       return kUnknownSize;
```

Comment:
I also lost a comment saying that this wasn't you're code, but perhaps good to adjust to the guidelines:
- https://llvm.org/docs/CodingStandards.html#don-t-use-else-after-a-return
- https://llvm.org/docs/CodingStandards.html#don-t-use-braces-on-simple-single-statement-bodies-of-if-else-loop-statements

Reply:
Thanks, that's very helpful. I got reviews regarding the coding style as well.